packages/@ama-sdk/schematics/README.md

```diff
@@ -64,19 +76,248 @@ If you use `Yarn2+`, you can use the following `scripts` in `package.json`:
 
 Use `generate` to (re)generate your SDK based on the content of `./swagger-spec.yaml` (make sure you have this file at the root of your project) and `upgrade:repository` to regenerate the structure of your project.
 
-### Java Client Core SDK
+### Supported features
```

Comment:
You should add a link to the supported feature targeting the OpenApi Documentation of the feature.

I also wonder if instead of listing the different features supported, it would not make more sense to list the ones that are not (and in progress)?
The listed you have listed can be part of example features